### ISSUE TITLE
inaccurate block placement 

### ISSUE THREAD

**Issue by 1cmf** [SoltauFintel/blockpuzzle#27]
I'm trying to figure out why on my Samsung Galaxy A52 5G with android 13 oneUI 5.1 the block placement even when I use a mouse or app I use my finger of where I use a stylus are about three blocks off and I can't get them to go anywhere near where I'm aiming them I will install reinstalled clear the cache gone to the thing clear the app cache partition I've reset the apps my display app everything and I can't get it to work right point me in a direction and I hope you the best I can![SmartSelect_20231016_063906_Settings.jpg](https://github.com/SoltauFintel/blockpuzzle/assets/81460320/597bbf27-aabb-4683-a1b1-3b0d2ac53a44)

**Comment by SoltauFintel** [comment]
see issue #23

**Comment by SoltauFintel** [comment]
I'm using Samsung A51 and for me it looks like we are having the same problem. (#23)
Restarting the phone the app works until ending the app.

**Comment by 1cmf** [comment]
Yes it does seem we were having the same problem mine does not resolve at all after reinstalling or restarting or force stopping the app ......I'm not sure what to do next with that other than to use my other old device that's rooted and it works perfectly fine ..... HOWERVER....thats not  a fix, that's just an avoidance that's all I can say ......I don't know how to go about trying to solve this problem without having to rewrite a API or something like that And I am not a programmer or anything close to that ..........In your opinion how should we proceed?
